docs/deploy.md

```diff
@@ -205,7 +205,7 @@ Currently, we test the following two container registries:
 
    ```console
    cf auth admin <cf-values.yml.cf-admin_password>
-   # or using yq: cf auth admin "$(yq -r '.cf_admin_password' ${TMP_DIR}/cf-values.yml)"
+   # or using yq: cf auth admin "$(yq r ${TMP_DIR}/cf-values.yml 'cf_admin_password')"
```

Comment:
interesting - this change actually makes this yq call go from working to not working for me.

I've installed `yq` via `brew install python-yq`, which `yq` are you using?

Reply:
ooooh interesting.  we have multiple versions of yq !?!?!

I use https://mikefarah.gitbook.io/yq/ (on linux v 3.1.1 ).  do you have a pre-3 version?  I think it may have had a more jq like syntax back then.

Reply:
If version mismatches like this exist, then maybe it would be better to skip it?  or use `cat tmp/cf-values.yml | grep '^cf_admin_password:' | awk -F': ' '{ print $2}'` which is [probably] more universal.

Reply:
@paulczar There are two main flavors of `yq`. One of them is [kislyuk/yq](https://github.com/kislyuk/yq) managed via Python `pip` and the other is [mikefarah/yq](https://github.com/mikefarah/yq) - a binary compiled from Golang. Unfortunately they use significantly different syntax. We should definitely be clear which `yq` we are using in any given context.